scripts/20200106_remove-custom-logo/delete-custom-logo-key.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */
+
+/*
+Delete unused customLogo key for all forms
+*/
+
+// Check total forms count
+db.getCollection('forms').count() 
+
+// Check number of forms with customLogo key
+db.getCollection('forms').find({ customLogo: { $exists: true } }).count()
+
+// Check number of forms with startPage.logo key
+db.getCollection('forms').find({ "startPage.logo.state": { $exists: true } }).count()
```

Comment:
can we also check how many forms have only the `customLogo` key but not `startPage.logo.state`?

Reply:
Based on my queries yesterday:

**Total Number of forms:** 179888

**Forms with `startPage.logo.state` defined:** 139912

Breaks down into:
_DEFAULT_: 112081
_NONE_: 8297
_CUSTOM_: 19534

`customLogo` may exist for such forms too, but is currently ignored as `startPage.logo` is defined

**Forms with `startPage.logo.state` undefined:**  39977

Breaks down into:
`customLogo` undefined: 36359 => Currently defaults to agency logo
`customLogo` defined: 3618 => Currently defaults to `customLogo`, but once the key is removed, will default to agency logo

Reply:
@liangyuanruo there are a sizeable number of forms (i.e. 36359) with neither `startPage.logo.state` nor `customLogo` as new forms are created without either key

Reply:
to clarify, this is because it's possible to have a form without a logo?

Reply:
*Old Code*
- If `startPage.logo.state` present, will use state defined therein
- If `startPage.logo.state` absent, will use state defined in `customLogo`. Note that `undefined` state for `customLogo` means that the default agency logo is to be shown
- New forms are created with `startPage.logo.state` absent and `customLogo` `undefined` and hence show the default agency logo

*New Code*
- If `startPage.logo.state` present, will use state defined therein
- Will ignore `customLogo`, although `customLogo` should have been removed from `form` documents
- If `startPage.logo.state` absent, default agency logo will be shown
- New forms are created with `startPage.logo.state` absent and hence show the default agency logo

Reply:
does that make sense @liangyuanruo